bitnami/haproxy/README.md

```diff
@@ -0,0 +1,408 @@
+| `pdb.minAvailable`                   | Minimum number/percentage of pods that should remain scheduled                            | `1`                     |
+| `pdb.maxUnavailable`                 | Maximum number/percentage of pods that may be made unavailable                            | `nil`                   |
+| `autoscaling.enabled`                | Enable Horizontal POD autoscaling for HAProxy                                             | `false`                 |
+| `autoscaling.minReplicas`            | Minimum number of HAProxy replicas                                                        | `1`                     |
+| `autoscaling.maxReplicas`            | Maximum number of HAProxy replicas                                                        | `11`                    |
+| `autoscaling.targetCPU`              | Target CPU utilization percentage                                                         | `50`                    |
+| `autoscaling.targetMemory`           | Target Memory utilization percentage                                                      | `50`                    |
+| `command`                            | Override default container command (useful when using custom images)                      | `[]`                    |
+| `args`                               | Override default container args (useful when using custom images)                         | `[]`                    |
+| `hostAliases`                        | haproxy pods host aliases                                                                 | `[]`                    |
+| `podLabels`                          | Extra labels for haproxy pods                                                             | `{}`                    |
+| `podAnnotations`                     | Annotations for haproxy pods                                                              | `{}`                    |
+| `podAffinityPreset`                  | Pod affinity preset. Ignored if `affinity` is set. Allowed values: `soft` or `hard`       | `""`                    |
+| `podAntiAffinityPreset`              | Pod anti-affinity preset. Ignored if `affinity` is set. Allowed values: `soft` or `hard`  | `soft`                  |
+| `nodeAffinityPreset.type`            | Node affinity preset type. Ignored if `affinity` is set. Allowed values: `soft` or `hard` | `""`                    |
+| `nodeAffinityPreset.key`             | Node label key to match. Ignored if `affinity` is set                                     | `""`                    |
+| `nodeAffinityPreset.values`          | Node label values to match. Ignored if `affinity` is set                                  | `[]`                    |
+| `configuration`                      | haproxy configuration                                                                     | `<dummy configuration>` |
```

Comment:
`configuration` looks maybe a bit too generic, maybe something like `haproxyConfigurationOverride` could be more specific?

Reply:
It would not be true as we don't have a default configuration in the container. The name `configuration` is what we ue in charts like JupyterHub, Consul, Cassandra, InfluxDB, RabbitMQ and others. I'd leave it like this.

Reply:
Ok!